Chutzpah/ExecutionProviders/NodeTestExecutionProvider.cs

```diff
@@ -113,7 +113,18 @@ private static string BuildRunnerArgs(TestOptions options, TestContext context,
             var testModeStr = testExecutionMode.ToString().ToLowerInvariant();
             var timeout = context.TestFileSettings.TestFileTimeout ?? options.TestFileTimeoutMilliseconds ?? Constants.DefaultTestFileTimeout;
             string inspectBrkArg = context.TestFileSettings.EngineOptions.NodeInspect ? "--inspect-brk" : "";
-            runnerArgs = string.Format("{0} \"{1}\" {2} {3} {4} {5} {6} {7} {8}",
+
+            var engineBrowserOptions = string.Empty;
+            if (context.TestFileSettings.BrowserArguments != null)
+            {
+                var matchingEntries = context.TestFileSettings.BrowserArguments.Where(x => x.Key.Equals(options.Engine.ToString(), StringComparison.OrdinalIgnoreCase));
+                if (matchingEntries.Any())
+                {
+                    engineBrowserOptions = matchingEntries.FirstOrDefault().Value;
```

Comment:
You can just do First here since you know it cant be null at this point

Reply:
fixed